### ISSUE TITLE
Extraterrestrial life proven by IFs

### ISSUE THREAD

**Issue by wverhagen** [PardeeCenterDU/IFs-Issues-Tracking#120]
We have a small display issue. At least I hope it is a display issue.
Net migration for the world is not at zero, meaning we are getting ET life in IFs, and in some years people leaving earth.
That would be a finding :-)

Not only net migrations but also migrantsin for the world and migrantsout for the world differences are not a hard zero. Issue found in 7.75 but also pertains to later versions

![image](https://user-images.githubusercontent.com/18350553/139476867-73996652-28b4-46ab-a826-f4b40426af93.png)

**Comment by PardeeCenterIFs** [comment]
I don't think we should be using the extended number of decimals for this kind of analysis, anyway the differences are smaller than 1 single person. I think it's a rounding issue.

**Comment by PardeeCenterIFs** [comment]
During the Modeling Pod it was discussed that this issue should be kept open as a Rounding issue, to be vigilant in case we find instances were more than 6 decimals are needed, Dave mentioned he had found the need in Health variables, but didn't remember exactly which.
If we find a good test example my first inclination would be to change the units of the variable, otherwise we would probably need to start double precision variables, which will not be trivial to implement across the entire model, both vb6 and .NET.
